ui-tests/src/test/java/org/openmrs/reference/page/AppointmentBlocksPage.java

```diff
@@ -131,6 +131,7 @@ public void clickOnleft() {
      */
     public void clickOnDay() {
         clickOn(DAY);
+        waiter.until(ExpectedConditions.elementToBeClickable(DAY_CLASS));
```

Comment:
Often the `click `on  `DAY_CLASS` is intercepted by the above element and this results into `ElementClickInterceptedException`. Invoking `elementToBeClickable() ` paves the way